src/Aquifer.API/Endpoints/Notifications/List/Endpoint.cs

```diff
@@ -151,82 +161,101 @@ private static async Task<IReadOnlyList<ResourceCommentNotificationData>> GetRes
         DbConnection dbConnection,
         int userId,
         DateTime minimumCommentCreatedDate,
+        IConfiguration cofiguration,
         CancellationToken ct)
     {
+        var userIds = GetNotifyUserIdsOnCommunityReviewerComment(cofiguration);
+        var isContainingCommunityReviewerComments = userIds is not null && userIds.Contains(userId);
+        const string communityReviewerCommentsSubquery = """
+                                                         OR EXISTS (
+                                                             SELECT NULL
+                                                             FROM CommentThreads ct2
+                                                             JOIN Comments c2 ON c2.ThreadId = ct2.Id
+                                                             WHERE  
+                                                                 c2.UserId IN (
+                                                                     SELECT Id FROM Users WHERE Role = 6
```

Comment:
Please use the constant injected via string interpolation.
```suggestion
                                                                     SELECT Id FROM Users WHERE Role = {UserRole.CommunityReviewer}
```

Reply:
Doesn't string interpolation make it slower?

Reply:
It's only building the string in this case, not doing parameterization of the SQL query.  So yes, it will be very slightly slower.  But not embedding constant values is worth that minor performance cost and also makes the code more readable.